python/pickle/PyROOT_readtests.py

```diff
@@ -107,12 +107,27 @@ def __doftest( self, d ):
    def test5ReadCustomTypes( self ):
       """Test reading PyROOT custom types"""
 
-      o = pickle.load( self.in1 )
-      self.assertEqual(o, [123,123.123] )
+      self.exp_pyroot = os.environ.get('EXP_PYROOT') == 'True'
+
+      p = pickle.load( self.in1 )
+      cp = cPickle.load( self.in2 )
+
+      if self.exp_pyroot:
+         import ctypes
```

Comment:
Can we add here a small comment saying why we test `ctypes` instead of `ROOT.Long` and `ROOT.Double`?

For example, if you see here:
https://github.com/root-project/roottest/commit/822fb6316a93453b2f4b1bab20a3c33475e70ef0#diff-7dd31c15ec3c7cc0abde12371883f7b9R240

You can use basically a similar comment (with reference to the cppyy 101 ticket included).

Reply:
We need to refer to both long and double here:
```
# Cppyy's Long and Double will be deprecated in favour of
# ctypes.c_long and ctypes.c_double, respectively
```